_maps/pubbystation.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`Virology Interior Airlock at Virology (90,110,2) failed to find a valid airlock to cyclelink with!`
<hr>
This one's pretty simple. It's caused by the 'cyclelink helpers' on the virology airlock doors not being able to handle corners.

Luckily, the airlock doesn't actually need the helpers at all since it's using the 'bolt one and open the other' airlock system, so it's safe to just remove the helpers.
![ezgif com-gif-maker](https://user-images.githubusercontent.com/57483089/169620116-ed14a73e-9288-46ea-91b7-936bd908fdf5.gif)